### PR TITLE
Added Simple Form and regenerated the Devise views so it could be used there 

### PR DESCRIPTION
**Pull request by obulkin**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -60,4 +60,7 @@ group :development, :test do
 
   # Use factory_girl in place of fixtures for generating test objects
   gem "factory_girl_rails", "~> 4.0"
-end
\ No newline at end of file
+end
+
+# Use Simple Form for easier form creation
+gem "simple_form"
```
Comment (lasley) [inline Gemfile]: https://robots.thoughtbot.com/no-newline-at-end-of-file

File: config/initializers/simple_form.rb [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+  # Whether attributes are required by default (or not). Default is true.
+  # config.required_by_default = true
+
+  # Tell browsers whether to use the native HTML5 validations (novalidate form option).
+  # These validations are enabled in SimpleForm's internal config but disabled by default
+  # in this configuration, which is recommended due to some quirks from different browsers.
+  # To stop SimpleForm from generating the novalidate option, enabling the HTML5 validations,
+  # change this configuration to true.
+  config.browser_validations = false
```
Comment (lasley) [inline config/initializers/simple_form.rb]: Might be worth enabling this (it's nice). Browser support for HTML5 validations is >90% at this point, so assuming the user is semi up-to-date you're good. Maybe team input?

http://caniuse.com/#feat=form-validation

File: Gemfile (hunk 1 above)
Comment (obulkin) [inline Gemfile]: http://www3.pictures.zimbio.com/mp/mA5SbP1W53Ex.gif. Thanks for the info. I'll stop deleting these.

File: config/initializers/simple_form.rb (hunk 2 above)
Comment (obulkin) [inline config/initializers/simple_form.rb]: Works for me. I'll ask the team about this general scenario.

Comment (obulkin) [inline config/initializers/simple_form.rb]: The team voted to allow any front-end code with >80% coverage in the US, so HTML5 validations should be good. I'll change this.

**Comment by AidanZeber** [comment]
Code reviewed. Aligned with changes after reviewing comments. Merging pull request.

**Pull request merged**